### DISCUSSION TITLE
Multiple authentication methods

### DISCUSSION THREAD

**Discussion by younes199511** [Kong/kong#7595] [Help] [answered]
Hello,

I have a use case where i need to setup JWT and Ldap  authentication plugins in Kong. I was able to do that successfully but the issue is that the APIs are accessible only with both methods simultaneously (this is not the behavior i'm looking for). I want to allow the two methods of  authentication  but be able to access the API by only one (JWT token or Ldap user:password). 

Is that possible with Kong ? I saw in the issue #590 published in 2016 that was not possible at that version of kong and i want to know if it has been implemented since. 
 

Best regards

**Comment by bungle** [comment, marked as answer]
@younes199511,

To make them OR you need to set both with config.anonymous and then set e.g. request termination plugin for anonymous consumer (or use acl plugin). Yes, we agree it is not intuitive.
